### PR TITLE
[TIMOB-20461] iOS: Events fired by Slide Over and Split Window not clearly documented

### PR DESCRIPTION
**Pull request by AngelkPetkov**
[JIRA](https://jira.appcelerator.org/browse/TIMOB-20461)

### CODE REVIEW

File: apidoc/Titanium/App/App.yml [hunk 1]
```diff
@@ -194,7 +194,9 @@ events:
         Available on iOS 4.0 and later.
         
         This event fires when the user leaves the application or for certain types of temporary interruptions 
-        such as a notification or incoming phone call.  
+        such as a notification or incoming phone call. If building an IPad application with split view enabled,
```
Comment (FokkeZB) [inline apidoc/Titanium/App/App.yml]: IPad should be iPad

File: apidoc/Titanium/App/App.yml [hunk 2]
```diff
@@ -194,7 +194,9 @@ events:
...
+        the event will be called ones the application is put in the background and a different split view
```
Comment (FokkeZB) [inline apidoc/Titanium/App/App.yml]: ones should be once

File: apidoc/Titanium/App/App.yml [hunk 3]
```diff
@@ -293,8 +295,9 @@ events:
     summary: Fired when the application returns to the foreground.
     description: |
         This event fires when the application enters the foreground with the `resume` event or 
-        returns to focus after a notification or incoming phone call.
-        
+        returns to focus after a notification or incoming phone call. If building an iPad application
```
Comment (FokkeZB) [inline apidoc/Titanium/App/App.yml]: "On iPad with split view enabled, this event will be called every time the application resized."

**Comment by AngelkPetkov** [comment]
Feedback addressed! :)

**Comment by hansemannn** [comment]
Approved!

**Pull request merged**